src/simplewallet/simplewallet.cpp

```diff
@@ -2343,21 +2334,17 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
     }
   }
 
-  size_t fake_outs_count;
-  if(local_args.size() > 0) {
-    size_t ring_size;
-    if(!epee::string_tools::get_xtype_from_string(ring_size, local_args[0]))
-    {
-      fake_outs_count = m_wallet->default_mixin();
-      if (fake_outs_count == 0)
-        fake_outs_count = DEFAULT_MIX;
-    }
-    else
-    {
-      fake_outs_count = ring_size - 1;
-      local_args.erase(local_args.begin());
+    size_t fake_outs_count;
+    if (local_args.size() > 0) {
+        size_t ring_size;
+        if (!epee::string_tools::get_xtype_from_string(ring_size, local_args[0])) {
```

Comment:
Could you please change this structure to:

```
fake_outs_count = DEFAULT_MIX;
if (epee::string_tools::get_xtype_from_string(ring_size, local_args[0])) {
    local_args.erase(local_args.begin());
}
```

For better code optimization/performance.